nuxt.config.js

```diff
@@ -34,6 +34,7 @@ export default {
     // https://go.nuxtjs.dev/axios
     '@nuxtjs/axios',
     '@nuxtjs/i18n',
+    '@nuxtjs/moment',
   ],
   i18n: {
```

Comment:
Can we move i18n to json files?

Reply:
Thanks bhai, I'll definitely go through this.